src/config/firebase.jsx

```diff
@@ -0,0 +1,20 @@
+
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+// import { getAnalytics } from "firebase/analytics";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyDFoTDCkven6e30fcSLZVW2YnvNet8GD3I",
+  authDomain: "teampick-99790.firebaseapp.com",
+  projectId: "teampick-99790",
+  storageBucket: "teampick-99790.appspot.com",
+  messagingSenderId: "493084048516",
+  appId: "1:493084048516:web:7eaaacc1319256a9fd96d1",
+  measurementId: "G-5CRPXGHCYF"
+};
```

Comment:
I was meaning to ask about this. Is it okay storing these keys here or should it be obfuscated somehow? Sorry for the dumb question!

Reply:
We did look it up and as far as we can tell... yes with the Firebase one. Right @auretswanepoel?